File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-persons-leahy-vetting_result.ts


import { GetPersonsLeahyVetting_Item } from './get-persons-leahy-vetting_item';
import { IGetPersonsLeahyVetting_Result } from './iget-persons-leahy-vetting_result';

export class GetPersonsLeahyVetting_Result implements IGetPersonsLeahyVetting_Result {
  
	public item?: GetPersonsLeahyVetting_Item;
  
	public ErrorMessages: string[] = null;
}

File: GTTS/INL.VettingService.Models/GetPersonsLeahyVetting_Result.cs
using System;
using System.Collections.Generic;
using System.Text;
using INL.Services.Models;

namespace INL.VettingService.Models
{
    public class GetPersonsLeahyVetting_Result : BaseResult, IGetPersonsLeahyVetting_Result
    {
        public GetPersonsLeahyVetting_Item item { get; set; }
    }
}
